src/pages/LoginScanStep.tsx

```diff
@@ -47,7 +84,7 @@ export default function LoginScanStep() {
               fontFamily: 'Poppins'
             }}
           >
-            Login with Learner Credential Wallet
+            Connect with Learner Credential Wallet
```

Comment:
```suggestion
            Login with Learner Credential Wallet
```
Leave 'Login' for now, we'll wordsmith later